sessiontxn/staleread/provider.go

```diff
@@ -93,7 +93,7 @@ func (p *StalenessTxnContextProvider) activateStaleTxn() error {
 		return err
 	}
 
-	txnScope := config.GetTxnScopeFromConfig()
+	txnScope := kv.GlobalTxnScope
```

Comment:
Why change?

Reply:
https://github.com/pingcap/tidb/pull/35877 changed some codes, this is to align with it.

Reply:
Got